epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -65,7 +65,9 @@
 @AutoService(Processor.class)
 public class EpoxyProcessor extends AbstractProcessor {
   private static final String GENERATED_CLASS_NAME_SUFFIX = "_";
-  private static TypeMirror epoxyModelType;
+  private static final String EPOXY_MODEL_TYPE = "com.airbnb.epoxy.EpoxyModel<?>";
```

Comment:
Instead of referencing the model class and annotation directly I stubbed them out.